### ISSUE TITLE
+ dark mode

### ISSUE THREAD

**Issue by tomByrer** [BuilderIO/vscode#1]
Neat tool!

The lack of dark mode is a bit of A11y issue for me; too much white screens can aggravate my concussion.  I had to install a "Dark Mode" browser exertion to help prolong the computer sessions I can do.

Also, I find there isn't enough contrast &/or line thickness for your current white mode.

**Comment by tomByrer** [comment]
closing: seems most of the issues should go to the fiddle, since this plugin is just a web view.  (I was wondering why the extention's codebase seemed so small!)
